src/cvode/cvode_impl.h

```diff
@@ -664,6 +665,11 @@ int cvDiagSetup_buildM(const sunrealtype fract, const sunrealtype uround,
 int cvDiagSolve_updateM(const sunrealtype r, N_Vector M);
 #endif
 
+/* Utility function to tell CVODE to free the user data.
+   This is used by the Python interfaces. */
+
+int CVodeSetOwnUserData(void* cvode_mem, sunbooleantype own_user_data);
```

Comment:
Prefix internal functions with `cv`
```suggestion
int cvSetOwnUserData(void* cvode_mem, sunbooleantype own_user_data);
```